packages/nuxt/src/app/composables/cookie.ts

```diff
@@ -58,7 +63,7 @@ export function useCookie<T = string | null | undefined> (name: string, _opts?:
   }
 
   if (import.meta.client) {
-    const channel = typeof BroadcastChannel === 'undefined' ? null : new BroadcastChannel(`nuxt:cookies:${name}`)
+    const channel = store || typeof BroadcastChannel === 'undefined' ? null : new BroadcastChannel(`nuxt:cookies:${name}`)
```

Comment:
Why was that line changed? I think the interface of a BroadcastChannel and the cookiestore don't align

Reply:
@manniL It is used only in the condition: Create `new BroadcastChannel` only if `cookieStore` doesn't exist and `BroadcastChannel` is supported. So `channel` can be an instance of `BroadcastChannel` or `null`.